### DISCUSSION TITLE
how verify fields that not render on dom?

### DISCUSSION THREAD

**Discussion by HZ-Zero** [react-hook-form/react-hook-form#9230] [Q&A] [answered]
I use react-window and useFormArray to improve form performance, but don't find api to validate all the field (just validate fields that render on the dom).
![image](https://user-images.githubusercontent.com/47170931/196113901-7939b24d-d278-4e66-865c-0089ef83dec3.png)

code: https://codesandbox.io/s/cool-sunset-e0isln?file=/src/index.jsx:1719-1813

**Comment by Moshyfawn** [comment]
Does this help?
https://react-hook-form.com/advanced-usage/#Workingwithvirtualizedlists

**Reply by HZ-Zero** [reply]
not really, It seems has the same problem that can't validate all the list.But any way I can validate all the list by myself (I can get all the value and rules.).I just want an accessible way。
![20221018142327_rec_](https://user-images.githubusercontent.com/47170931/196351769-8af71792-9277-4f71-80d2-ece6a27a089a.gif)

**Comment by Moshyfawn** [comment, marked as answer]
K, so, RHF doesn't support unmounted inputs validation at field level; you'll have to opt-into using validation schema instead. See #7178 for more info

**Reply by HZ-Zero** [reply]
Ok, I got it , thank you~
